content/docs/introduction/central.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Open mSupply Central Server"
+description = "Open mSupply central server functionality"
+date = 2022-06-10T11:38:00+00:00
+updated = 2022-06-10T11:38:00+00:00
+draft = false
+weight = 8
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = true
++++
+
+Open mSupply central server is a special site that allows for configurations of subsets of central data. Please see [requirements](/docs/introduction/requirements/#open-msupply-requirements) section for extra info.
+
+## What is the Open mSupply central server
+
+In essence it's just another Open mSupply instance. However, the differences from a typical remote site are:
+
+- Only one instance of the Open mSupply central server will exist in the Open mSupply system
+- It will be configured by our support staff and needs to be available on the World Wide Web (typically as a cloud server, but can also be hosted in country)
```

Comment:
:nit the phrase 'in-country' here is a specific thing, and not just 'in a country' which everything is, kinda.
Alternatively, 'hosted locally' or similar.
```suggestion
- It will be configured by our support staff and needs to be available on the World Wide Web (typically as a cloud server, but can also be hosted in-country)
```